scripts/virus-scan-s3-bucket.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+"""Virus scan the contents of an S3 bucket, optionally from a given date onwards.
+Requires a clamd backend to be provided (e.g. https://hub.docker.com/r/mkodockx/docker-clamav/)
+
+You'll need to configure the required AWS environment variables, normally either AWS_PROFILE, or
+AWS_ACCESS_KEY_ID & AWS_SECRET_ACCESS_KEY.
+
+Example:
+    ./scripts/virus-scan-s3-bucket --since 2018-01-01T00:00:00Z digitalmarketplace-documents-production-production
+"""
+
+import argparse
```

Comment:
Not a fan of docopt like the rest of our scripts? I must say I often find it quite frustrating so interested to see how argparse plays out for you

Reply:
That's docopt I find it frustrating by the way, not argparse (which I've never used)

Reply:
I really don't like docopt for a number of reasons. I think argparse is cleaner to use, and it's faster at the command line IMO.